cluster-autoscaler/main.go

```diff
@@ -487,6 +488,7 @@ func buildAutoscaler(debuggingSnapshotter debuggingsnapshot.DebuggingSnapshotter
 
 	opts.Processors = ca_processors.DefaultProcessors(autoscalingOptions)
 	opts.Processors.TemplateNodeInfoProvider = nodeinfosprovider.NewDefaultTemplateNodeInfoProvider(nodeInfoCacheExpireTime, *forceDaemonSets)
+	opts.Processors.BinpackingLimiter = binpacking.NewTimeLimiter(*maxInactivityTimeFlag/2, opts.Processors.BinpackingLimiter)
```

Comment:
`maxInactivityTimeFlag/2` sounds like a reasonable default, but let's have a separate flag for this.

Reply:
Introduced ```max-binpacking-time```